examples/mongodb/babel.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  // See https://babeljs.io/docs/en/babel-preset-env#targets
+  presets: [['@babel/preset-env', {targets: {node: 'current'}}]],
```

Comment:
```suggestion
  presets: [['@babel/preset-env', {targets: {node: 6}}]],
```
should fix CI.
Node 6 is the minimum Node version Jest supports.